src/border_agent/border_agent.cpp

```diff
@@ -111,13 +111,19 @@ struct StateBitmap
     uint32_t mAvailability : 2;
     uint32_t mBbrIsActive : 1;
     uint32_t mBbrIsPrimary : 1;
+#if OTBR_EPSKC_ENABLE
+    uint32_t mEPSKcSupported : 1;
+#endif
 
     StateBitmap(void)
         : mConnectionMode(0)
         , mThreadIfStatus(0)
         , mAvailability(0)
         , mBbrIsActive(0)
         , mBbrIsPrimary(0)
+#if OTBR_EPSKC_ENABLE
+        , mEPSKcSupported(1)
+#endif
```

Comment:
```suggestion
        , mEPSKcSupported(OTBR_EPSKC_ENABLE)
```

Reply:
Accept. also need to add definition for OTBR_ENABLE_EPSK in hpp file